src/java/org/apache/cassandra/config/Config.java

```diff
@@ -519,15 +526,15 @@
      * Time in milliseconds after a warning will be emitted to the log and to the client that a UDF runs too long.
      * (Only valid, if user_defined_functions_threads_enabled==true)
      */
-    //No need of unit conversion as this parameter is not exposed in the yaml file
```

Comment:
all fields in this file are exposed to YAML, so renamed back to the old name and did a conversion to support the SmallestDurationMilliseconds and old Long (as millis)